libshaderc_util/include/libshaderc_util/counting_includer.h

```diff
@@ -24,7 +24,13 @@ namespace shaderc_util {
 // An Includer that counts how many #include directives it saw.
 class CountingIncluder : public glslang::TShader::Includer {
  public:
-  CountingIncluder() : num_include_directives_{0} {}
+  // This is not done in the intiailizer list, and done as .store(0)
```

Comment:
This doesn't quite parse.
Needs a period at the end of this line.
Also, the "and done as .store(0)" seems like it's part of the negative statement.
Suggest: " Done as .store(0) instead of in the initializer list, for the following reasons:"